ENIGMAsystem/SHELL/Graphics_Systems/General/GSsprite.cpp

```diff
@@ -70,7 +70,7 @@ using namespace enigma_user;
 namespace enigma
 {
 
-void draw_sprite_pos_raw(const enigma::sprite* spr2d, int subimg, gs_scalar x1, gs_scalar y1, gs_scalar x2, gs_scalar y2, gs_scalar x3, gs_scalar y3, gs_scalar x4, gs_scalar y4, gs_scalar color, gs_scalar alpha)
+void draw_sprite_pos_raw(const enigma::sprite *const spr2d, int subimg, gs_scalar x1, gs_scalar y1, gs_scalar x2, gs_scalar y2, gs_scalar x3, gs_scalar y3, gs_scalar x4, gs_scalar y4, gs_scalar color, gs_scalar alpha)
```

Comment:
Why bother making the pointers themselves const? Were we getting link errors?

Reply:
Here's the thing, the ASAN error I got from loading the png with the libpng extension disabled read as "binding null pointer to const* type" which I mistook as the function signature being different than what the macro actually returns. That said, this is a harmless change. Since that's the type the get sprite macro actually declares, and the function doesn't need to change the pointer, we may as well keep this change.